cmd/config.go

```diff
@@ -187,5 +190,12 @@ func initConfig(objAPI ObjectLayer) error {
 		return err
 	}
 
+	secretKey := globalServerConfig.GetCredential().SecretKey
+
+	// Migrates backend config and encrypts it.
+	if err := migrateMinioSysConfigAndEncrypt(objAPI, secretKey); err != nil {
```

Comment:
Is `SecretKey` mandatory, otherwise you probably want to check if it is set?

Reply:
IT is not mandatory, I shall add it as LikelyEncrypt() to the function name.